include/vigra/random_forest/rf_common.hxx

```diff
@@ -490,7 +495,7 @@ class RandomForestOptions
 
     /** How many trees to create?
      *
-     * <br> Default: 255.
+     * <br> Default: 256.
```

Comment:
If there is a mismatch between documentation and implementation here, please change the implementation: we want 255 trees because their response fits into 8 bits.